solanum-tuberosums/djangazon
# User can add product category to product when selling.

## Associated Ticket
[User can select product category](https://github.com/solanum-tuberosums/djangazon/issues/10

## Description 
- Added ProductCategory dropdown selector to the SellProduct form 
- Added __str__() method to ProductCategory class
### To Test:
- Add 1 or 2 ProductCategories to your local database (through DB browser OR python shell)
- Register/login and navigate to [localhost/sell](http://localhost:8000/sell)
- Make sure the user can see the added categories in the dropdown


## All Tests Pass
N/A
